packages/gatsby/src/query/types.ts

```diff
@@ -28,7 +28,7 @@ export interface IGraphQLRunnerStatResults {
   uniqueSorts: number
 }
 
-export type PageContext = any
+export type PageContext = unknown
```

Comment:
This is a breaking change, because it means anything that uses context will need to narrow the type

Reply:
This is why the tests are failing

Reply:
@blainekasten Is that true? I don’t understand typescript well enough to know this

Reply:
I'll switch back to `any`, hold on

Reply:
/shrug I didn't know that :) `any` is fine then :)